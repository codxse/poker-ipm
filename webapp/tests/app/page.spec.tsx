import { render, screen } from '@testing-library/react'
import Home from '@app/page'
import '@testing-library/jest-dom'

describe('Home', () => {
  it('renders homepage unchanged', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
  })

  it('renders a heading', () => {
    render(<Home />)

    const heading = screen.getByRole('heading', {
      name: /Hello world!/i,
    })

    expect(heading).toBeInTheDocument()
  })
})
