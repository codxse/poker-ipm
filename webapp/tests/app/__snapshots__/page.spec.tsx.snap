// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders homepage unchanged 1`] = `
<div>
  <main
    class="flex min-h-screen flex-col items-center justify-between p-24"
  >
    <h1>
      Hello world!
    </h1>
  </main>
</div>
`;
